resources/views/intro.twig

```diff
@@ -2,38 +2,14 @@
 
 {% block content %}
   <div class="product_collection">
+    
     <h1>Welcome to {{ product_collection.post_title }}!</h1>
-
-
+	  
 	  <ul class="product_collection-category-list">
 	    {% for category in product_collection.get_field('categories') %}
-	      <li><a href="{{product_collection.slug}}/{{category.name}}">{{category.name}}</a></li>
+	      <li><a href="{{product_collection.slug}}/{{category.name|lower}}">{{category.name}}</a></li>
```

Comment:
Does `{{category.name|lower}}` this work in all instances? What if the category name was "Pain Au Chocolat" wouldn't that make the URL "pain%20au%20chocolat"?